torchvision/datasets/utils.py

```diff
@@ -42,6 +42,23 @@ def check_integrity(fpath: str, md5: Optional[str] = None) -> bool:
     return check_md5(fpath, md5)
 
 
+def _get_redirect_url(url: str, hop_idx: int = 0) -> str:
+    import requests
+
+    _MAX_HOPS = 10
```

Comment:
1. Why did you prefix this with an underscore?
2. IMO we should add this as a default parameter in case someone wants to use a stricter setting.

Reply:
Hey @pmeier,
Referring to you questions:  
> prefix this with an underscore
The code was adapted from the class method, so it inherited a class field as is.

> add this as a default parameter 
Makes perfect sense, I will update the PR.